### PR TITLE
Domain restore documentation

### PR DESCRIPTION
**Pull request by kntsoriano**
This adds the support documentation for domain restore.

To be more specific, this adds the following:
* Add page on how to restore a domain.
* Update specific TLD docs to link to the restoration page.
* Update "What happens when a domain expires page?" to include domain restoration.

### CODE REVIEW

File: content/files/restore-domain-form.png
Comment (OleMchls) [inline content/files/restore-domain-form.png:1]: We should not display our ops e-mail address here

**Review by OleMchls — APPROVED** [review]
Please have @itsalyse review the copy :)

File: content/articles/restoring-domain.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Domains
+---
+
+# Restoring a Domain
+
+To reactivate your domain that has entered the [redemption period](/articles/what-happens-when-domain-expires/#redemption-period), it has to be restored. After completion, your domain's registration will be extended by the minimum renewal period (usually 1 year).
+
+<warning>
+The registrar applies a mandatory restore fee.
+The final price of the restore will be the <strong>restore fee plus the standard renewal fee</strong>.
```
Comment (stephanoodle) [inline content/articles/restoring-domain.md:14]: We should mention that the renewal pricing will differ for each TLD.

File: content/articles/restoring-domain.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ![Restore domain link](/files/restore-domain.png)
+
+1.  Confirm the price and click the <label>Restore Domain</label> button.
+
+    ![Restore domain form](/files/restore-domain-form.png)
+
+<info>
+The total restore price will be held on your card immediately.
+Your card will be charged once the restoration completes.
+If the restoration fails, funds will be released.
```
Comment (stephanoodle) [inline content/articles/restoring-domain.md:32]: We need to tell the customers that they will receive an email once the restore completes, or an email if it fails.

**Review by stephanoodle — CHANGES_REQUESTED** [review]

**Comment by SarahCaminiti1** [comment]
I've added Alyse and removed myself as I'll be OOO

**Comment by kntsoriano** [comment]
@SarahCaminiti1 Thanks for letting me know. I'll transfer this to a google doc.

**Comment by kntsoriano** [comment]
Closing this for now. Created the google doc: https://docs.google.com/document/d/1oLXT8WWirT_PzSgo4T3N_ExFr8v0Gd24ArvUNx6d6lE/edit

**Comment by stephanoodle** [comment]
> Closing this for now. Created the google doc: https://docs.google.com/document/d/1oLXT8WWirT_PzSgo4T3N_ExFr8v0Gd24ArvUNx6d6lE/edit

FYI this doc looks good to me.

**Pull request closed without merging**